docs/stacks-guide-gcp.md

```diff
@@ -137,38 +145,208 @@ The name of the `Provider` resource in the file above is `gcp-provider`;
 we'll use the name `gcp-provider` to refer to this provider when we
 configure and set up other Crossplane resources.
 
-## Set Up Provider Resources
+## Set Up Network Resources
```

Comment:
Got it. I felt like `Provider Resources` is too general for that step. I think, in the next steps what we do is still creating `Provider Resources`. So, I wanted to differentiate this. No strong opinion though, it could be both ways.

Reply:
kk let's try it your way for now and reevaluate after we do some testing of the guide